language-family/.github/workflows/go-get-update.yml

```diff
@@ -0,0 +1,47 @@
+name: Go Get Update
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Once a day at midnight
+  workflow_dispatch: {}
+
+jobs:
+  update:
+    name: Go Get Update
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18.x
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - shell: bash
+      run: |
+        go get -u
```

Comment:
I just tried running this in the Go language family buildpack repo. The result was:
```
go: no package to get in current directory
```

This is because `go get` [fails for packages that don't have a `main()`](https://github.com/golang/go/issues/52179). 

So, I don't think we can/should add this automation for language family buildpacks, since they don't have `main()` functions.

Reply:
I've removed the language family workflow.

I have a vague memory of running `go get -u` on the `cpython` language family buildpack. Weird. https://github.com/paketo-buildpacks/python/pull/488

Reply:
I think you may want `go get -u all` instead. Running that on the Go language family buildpack yields results.

Reply:
Nice find @ForestEckhardt, I've updated the PR